core/controllers/suggestion.py

```diff
@@ -640,14 +1054,22 @@ def _get_target_id_to_skill_opportunity_dict(suggestions):
     }
 
     for opp_id, skill in opportunity_id_to_skill.items():
-        if skill is not None:
-            opportunity_id_to_opportunity_dict[opp_id]['skill_rubrics'] = [
-                rubric.to_dict() for rubric in skill.rubrics]
+        opportunity_dict = opportunity_id_to_opportunity_dict[opp_id]
+        if skill is not None and opportunity_dict is not None:
+            # Here we use MyPy ignore because we are adding a new
```

Comment:
Would it be more correct to give the resulting dict a different named type? Otherwise I can see issues down the line with not knowing which type of opportunity dict it is.

Reply:
A new type is added PTAL!